src/wmnds/components/content-tiles/map/_map.scss

```
@@ -0,0 +1,3 @@
+img {
```

GIve the img a class such as 
```suggestion
.wmnds-content-tiles__img {
```
As we don't want to risk targeting all images.
If you look at the header, you can see this scss img style is making the header super big.